source/_integrations/loqed.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: Loqed Touch Smart Lock
+description: Instructions on how to integrate a Loqed Touch Smart Lock
+ha_category:
+  - Lock
+ha_release: 2022.7
+ha_iot_class: Local Push
+ha_codeowners:
+  - "@cpolhout"
+ha_domain: loqed
+ha_platforms:
+  - lock
+  - sensor
+ha_config_flow: true
+ha_dhcp: true
+ha_integration_type: integration
+---
+
+Integrate your LOQED Touch Smart Lock with Home Assistant. The lock instantly notifies Home Assistant of a lock state change (no polling), and you can change the lock state yourself.
+
+## Features
+
+This integration supports:
+
+- Send real-time (!) status changes of the lock (open, unlock, lock)
+- Send battery status updates
+- Change the lock state (open, unlock, lock).
+  - IMPORTANT: The Home Assistant Lovelace standard card has only support for unlock and lock. By leveraging the lock-service: lock.open you can connect any button or action to opening the lock.
+  - Only if your lock has a fixed knob on the outside of your door, you can use the “open” lock state. If you do not have this (thus you have a handle on the outside of your door that you can push down), this command will behave as if the unlock command is sent.
+- Future: change certain settings on the LOQED Touch Smart Lock
```

Comment:
Future? As in? You can't right now? If that is the case, let's remove it.

Reply:
It's indeed future functionality. Removed it from the docs